tests/contracts_negative/SquareRoot.fz.expected_err

```diff
@@ -16,9 +16,9 @@ fuzion.runtime.pre_fault.cause#1: $MODULE/eff/fallible.fz:35:6:
 fuzion.runtime.precondition_fault#1: $MODULE/fuzion/runtime/pre_fault.fz:58:52:
 public precondition_fault(msg String) => pre_fault.cause msg
 ---------------------------------------------------^^^^^
-SquareRoot.#pre1_sqrt#1: --CURDIR--/SquareRoot.fz:30:9:
+SquareRoot.#pre1_sqrt#1: --CURDIR--/SquareRoot.fz:30:7:
       a ≥ 0
---------^^^
+------^^^^^^^
```

Comment:
sourcerange seems to be wrong? (too long)

Reply:
Created an issue for this: #3292